pages/learn/glossary.md

```diff
@@ -14,6 +14,12 @@ If you encounter a term you wish were defined here, please feel free to [file an
 
 The entries on this page can be linked to via anchor links (e.g. `https://json-schema.org/learn/glossary.html#vocabulary`) when sharing a definition with others.
 
+### compound schema resource
+
+A compound schema resource is a JSON document which has multiple embedded [JSON schema resources](#schema-resource). It is important to note that for a schema to be embeddable it must define a `$id` keyword, which is used to hold its unique identifier. These embedded schema resources collectively define various aspects or features of the overall schema.
+
+Compound schema resource is vital when working with multiple schema resources. By [bundling](../understanding-json-schema/structuring#bundlingbundling) these schema resources into a single schema document, distribution is made easier.
```

Comment:
> Compound schema resource is vital when working with multiple schema resources.

I'm not sure this is true.  A system can happily work with every resource in its own document.  Ideally, this is the way it should be done, but that's not always practical.

Also bundling, as described by that link, isn't the only reason to embed resources.  In an OpenAPI document block, each schema could have its own identifier, making it a distinct resource which could be `$ref`'d from anywhere.